### DISCUSSION TITLE
v 6.1.17 / Auto-Budgeting Issue: Seeking Community Input and Feedback

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#8976] [Firefly III]
Dear all,

I hope this message finds you well. I wanted to bring to your attention an issue I've encountered, and I'm curious if others may have experienced something similar.

The issue pertains to auto-budgeting in Firefly III:

The option "Set a fixed amount every period" functions correctly.
However, both "Add an amount every period" and "Add an amount every period and correct for overspending" do not accumulate the budget amount with the remaining from the previous period; instead, they behave similarly to the first option.
I am currently on a monthly budget cycle, and I can confirm that the cron job is operating correctly.

Additionally, I've noticed that in the total available budget, the remaining budget consistently displays as 0. I mention this in case it may be related to the issue at hand.

I appreciate any insights or feedback you may have on this matter.

Best regards,

**Comment by JC5** [comment]
> do not accumulate the budget amount with the remaining from the previous period; instead, they behave similarly to the first option.

I can't replicate this problem. It works fine for me.

**Reply by ghost** [reply]
Thanks you for your feedback :)
I will investigate further on my end
